app/models/user.rb

```diff
@@ -10,9 +10,10 @@ class User < ActiveRecord::Base
   has_many :repeats, dependent: :destroy
 
   validates :name, presence: true, length: {maximum: 100}
-  validates :role, presence: true, length: {maximum: 50}
 
-  def is_admin?
-    role == "admin"
+  before_create{self.role ||= Settings.user_roles[:normal]}
+
+  Settings.user_roles.each do |key,user_role|
```

Comment:
if you don't use `key`, you can use `_` to replace it. And you miss a space after `key,`